src/header/js/header-menus.js

```diff
@@ -44,7 +44,7 @@ export default ({variant = 'desktop'}) => {
 	/* ====================================================================== */
 
 	const createNavLink = (link) => {
-		const $link = $('button',
+		const $link = $('a',
```

Comment:
sometimes it should be a button sometimes it should be an anchor. Just key off the href. 

```
		let $link;

		if (link.props.href) {
			$link = $('a',
				{class: `${prefix}-${link.class}`, href: link.props.href},
				link.icon || "",
				link.label
			);
		} else {
			$link = $('button',
				{class: `${prefix}-${link.class}`},
				link.icon || "",
				link.label
			);
		}
```

Reply:
I made this change in a recent pr, FYI